client/state/stats/lists/utils.js

```diff
@@ -599,8 +599,12 @@ export const normalizers = {
 		}
 
 		const { startOf } = rangeOfPeriod( query.period, query.date );
-		const searchTerms = get( data, [ 'days', startOf, 'search_terms' ], [] );
-		const encryptedSearchTerms = get( data, [ 'days', startOf, 'encrypted_search_terms' ], false );
+		const dataPath = query.summarize ? [ 'summary', 'search_terms' ] : [ 'days', startOf, 'search_terms' ];
+
+		const searchTerms = get( data, dataPath, [] );
+		dataPath.pop();
+		dataPath.push( 'encrypted_search_terms' );
+		const encryptedSearchTerms = get( data, dataPath, false );
```

Comment:
could we rewrite this like this to avoid the pop/push ? 

```js
const dataPath = query.summarize ? [ 'summary' ] : [ 'days', startOf ];
const searchTerms = get( data, dataPath.concat( ['search_terms'] ), [] );
const encryptedSearchTerms = get( data, dataPath.concat( ['encrypted_search_terms'] ), false );
```